src/main/java/sg/ncl/MainController.java

```diff
@@ -1920,7 +1929,7 @@ public String removeMember(@PathVariable String teamId, @PathVariable String use
 //    @RequestMapping("/team_profile/{teamId}/remove_experiment/{expId}")
 //    public String removeExperimentFromTeamProfile(@PathVariable Integer teamId, @PathVariable Integer expId, Model model, HttpSession session) {
 //        // remove experiment
-//        // TODO check userid is indeed the experiment owner or team owner
+//        // TO DO check userid is indeed the experiment owner or team owner
 //        // ensure experiment is stopped first
 //        if (experimentManager.removeExperiment(getSessionIdOfLoggedInUser(session), expId) == true) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)